src/react/components/pages/homepage/homePage.tsx

```diff
@@ -185,8 +188,11 @@ export default class HomePage extends React.Component<IHomePageProps, IHomePageS
 
         this.props.applicationActions.ensureSecurityToken(project);
 
+        if (project.lastVisitedAssetId !== undefined) {
```

Comment:
I don't think anything specific to a given asset image/video should leak up into this part of the implementation.  This feels like something that should be handled by the `ImportService` that you are calling below.